app/views/shared/_beta_banner.html.haml

```diff
@@ -0,0 +1,6 @@
+.flash.notice
+  .small This service is still in development, so for now only lists jobs in select schools in Cambridgeshire and the North East. When it launches, the full service will list jobs in schools throughout England.
+  .small
+    Interested in listing jobs at your school? Email
+    =mail_to 'isobel.croot@digital.education.gov.uk'
```

Comment:
Can we extract this to an environment variable and reuse so it's not stored in the codebase.